plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -962,6 +982,61 @@ class FossId internal constructor(
                         requests += async {
                             service.markAsIdentified(config.user, config.apiKey, scanCode, path, false)
                         }
+
+                        val filteredSnippetChoicesByPath = snippetChoices.filter {
+                            it.given.sourceLocation.path == path
+                        }
+
+                        val relevantSnippetChoices = filteredSnippetChoicesByPath.filter {
+                            it.choice.reason == SnippetChoiceReason.ORIGINAL_FINDING
+                        }
+                        relevantSnippetChoices.forEach { filteredSnippetChoice ->
```

Comment:
Add empty line above.
You could also merge the filters like:
```kotlin
snippetChoices.filter {
  it.given.sourceLocation.path == path && it.choice.reason == SnippetChoiceReason.ORIGINAL_FINDING
}.forEach { snippetChoice ->
  ...
}
```

Reply:
The filters cannot be merged as the variable `relevantSnippetChoices` is also used a bit later:

```
val relevantChoiceCount = relevantSnippetChoices.size
```

Reply:
If `filteredSnippetChoicesByPath` is not used you could at least merge the two filter conditions but keep the `releavntSnippetChoices` variable.